docs/source/advanced/authentication.mdx

```diff
@@ -74,4 +74,50 @@ val apolloClient = ApolloClient.Builder()
     .build()
 ```
 
+### Updating your WebSocket token
+
+When using authenticated subscriptions over WebSockets, you might want to refresh your token. Because it was invalidated or simply because it expired.
+
+> Note: if your user logged out, you might want to consider creating a new `ApolloClient` instead to avoid sharing cache and/or other state between users.
```

Comment:
```suggestion
> Note: if your user logged out, you might want to consider creating a new `ApolloClient` instead of avoiding sharing cache and/or other state between users.
```

Reply:
I'll rephrase a bit. I wanted to put the emphasis on the fact that caches do need to be separated.

Reply:
Updated with 

```
> Note: if your user logged out, you might want to consider creating a new `ApolloClient` instead. This will make sure that all other state besides the WebSocket, like cache for an example, is new and not shared amongst different users.
```

Let me know what you think

Reply:
Yes, it is clearer 👍